openedx/core/djangoapps/user_api/accounts/api.py

```diff
@@ -243,17 +243,21 @@ def _validate_name_change(user_profile, data, field_errors):
         return None
 
     old_name = user_profile.name
+    new_name = data['name']
+
+    if old_name == new_name:
+        return None
```

Comment:
Does return `None` here means the name is valid?

Reply:
Adding a test case now. Yeah, this means the change is valid, and will also prevent it from being added to `old_names` in profile metadata.